lib/litmus_paper/dependency/base.rb

```diff
@@ -0,0 +1,16 @@
+module LitmusPaper
+  module Dependency
+    class Base
+      attr_reader :available
+
+      def available?
+        return @available unless @available.nil?
```

Comment:
So doing some research, based on where litmus actually creates the dependency objects ([in service.rb](https://github.com/braintree/litmus_paper/blob/c237b4d82f3eda6ad0867edcc8c351f07b6e694c/lib/litmus_paper/service.rb#L27-L29)), and that services are a part of the litmus config, it just calls `dependency.available?` each time a request is made, without creating new objects.